### PR TITLE
Sample for Typescript.

### PR DESCRIPTION
**Pull request by artursouza**
Addresses issue #18

### CODE REVIEW

File: hello-typescript/app.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+app.get('/order', (_req, res) => {
+    var get = new dapr.dapr_pb.GetStateRequest();
+    get.setStoreName(stateStoreName)
+    get.setKey("order");
+    client.getState(get, (error, response) => {
+        if (error) {
+            console.log(error);
+            res.status(500).send({message: error});
+        } else {
+            console.log('Got!');
```
Comment (msfussell) [inline hello-typescript/app.ts:49]: Should this not read "console.log('Found order!');
preferably with the order value

File: hello-typescript/README.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+== APP == Got a new order! Order ID: 41
+```
+
+## Step 6 - Code Changes
+
+In this sample, the project is setup in a way that code changes do not require the app to be restarted. Make a simple code change and see how it is loaded without the need to restart Dapr runtime or the app. The following log message can be modified and a new GET request made to reproduce this feature.
+
+```ts
+            console.log('Got!');
+```
```
Comment (msfussell) [inline hello-typescript/README.md:211]: console.log('Found order ID xx');

File: hello-typescript/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "version": "1.0.0",
+  "main": "app.ts",
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc",
+    "dev": "ts-node-dev --inspect --ignore-watch node_modules app.ts",
+    "start": "dapr run --app-id=nodeapp --dapr-http-port=3500 --app-port=3000 -- ts-node-dev --inspect --ignore-watch node_modules app.ts"
+  },
+  "dependencies": {
+    "dapr-client": "^0.7.0-preview01",
```
Comment (msfussell) [inline hello-typescript/package.json:12]: Can you update/test this against 0.11.0 and change this file.

File: hello-typescript/package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "express": "^4.17.1"
```
Comment (msfussell) [inline hello-typescript/package.json:13]: Is this version of express correct?

**Review by msfussell — COMMENTED** [review]
Reviewed and left some comments

File: hello-typescript/app.ts (hunk 1 above)
Comment (artursouza) [reply hello-typescript/app.ts:49]: I changed the log message. Showing the order would require parsing the JSON. I kept it simple instead.

File: hello-typescript/README.md (hunk 2 above)
Comment (artursouza) [reply hello-typescript/README.md:211]: Used the same message as in the code above.

File: hello-typescript/package.json (hunk 3 above)
Comment (artursouza) [reply hello-typescript/package.json:12]: This is the Javascript SDK version, not runtime.

File: hello-typescript/package.json (hunk 4 above)
Comment (artursouza) [reply hello-typescript/package.json:13]: I saw the PR on express version. Yes, this example is using 4.17 and is working :)

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**